Indonesian/botol-biru/minum.md

```diff
@@ -0,0 +1,5 @@
+Anda baru sadar bahwa anda seorang pembunuh, anda telah membunuh sebuah sekelompok ilmuan yang sedang mencari obat untuk kanker. 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.